todo_app/data/trello_items.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import os
+
+LIST_ID = os.getenv("TRELLO_LIST_ID")
```

Comment:
This works nicely here, but pre-loading these may bite you in the next module so worth keeping an eye out for it. 

The reason it could be a problem is that these lines of code run when this file is first _imported_. That works perfectly right now, but:
* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a better solution is one of:
* The easiest here, just load the environment variables when they're needed, e.g. in `get_items`:
```python
    response = requests.request(
        "GET",
        f"{TRELLO_API_URL}/lists/{os.getenv("TRELLO_LIST_ID")}/cards",
        params=API_PARAMS,
    )
```
* Personally I'd be lazy and go with the above here, but on a real codebase we'd probably prefer a more classic OOP solution; wrap these functions in a class, and load the environment variables in a constructor, e.g. something like:
```python
class TrelloService():

    def __init__(self):
        self.LIST_ID = os.getenv('TRELLO_LIST_ID')
        ...

    def get_items(self):
        response = requests.request(
            "GET",
            f"{TRELLO_API_URL}/lists/{self.LIST_ID}/cards",
            params=API_PARAMS,
        )
        ...
```

Reply:
for the second approach, when would you instantiate the class?
if we do it in the same file, would we not run into the same issue?

Reply:
Great question - we'll do it in `app.py` which absolutely does cause the same issues... _except_ that Flask has a standard solution for this!

Instead of defining the `app` variable at the top level, we'll define a `create_app` function which returns the `app` - that way the TrelloService class can be instantiated within the function, and our tests can only create the app when they're actually ready 😄